threadpool/core/src/main/java/cn/hippo4j/core/handler/DynamicThreadPoolBannerHandler.java

```diff
@@ -70,7 +70,7 @@ private void printBanner() {
             while (padding.length() < STRAP_LINE_SIZE - (bannerVersion.length() + DYNAMIC_THREAD_POOL.length())) {
                 padding.append(" ");
             }
-            System.out.println(AnsiOutput.toString(banner, AnsiColor.GREEN, DYNAMIC_THREAD_POOL, AnsiColor.DEFAULT,
+            log.info(AnsiOutput.toString("\n", banner, AnsiColor.GREEN, DYNAMIC_THREAD_POOL, AnsiColor.DEFAULT,
```

Comment:
Is the printing format correct after adjustment?

Reply:
Yes, I have conducted tests, and the print format is correct. Just like this: 

<img width="561" alt="image" src="https://github.com/opengoofy/hippo4j/assets/22118976/5f297f5b-7578-45ba-911e-af8b8d1ee391">